src/main-app.js

```diff
@@ -140,11 +159,47 @@ export class MainApplication extends LitElement {
       bottom: 16px;
       right: 16px;
     }
+
+    .btn-scan {
+      fill: #018786;
+    }
+
+    .disabled-scan {
+      fill: lightgray;
+    }
+
+    .div-bg-scan {
```

Comment:
we also use similar CSS for centering the other background icon. Can't we use a class instead then?

Reply:
Yeah, they are similar, but where's the best place to export this class? Create a new js file?